pages/memorials.vue

```diff
@@ -0,0 +1,422 @@
+<template>
+  <div id="map-wrapper" class="flex flex-col h-full w-full relative">
+    <loading-message v-if="loading" />
+    <div v-else>
+      <!-- Filters -->
+      <div class="absolute flex flex-col w-full h-full">
+        <div
+          class="xs:w-11/12 sm:w-1/2 lg:w-2/5 xl:w-1/3 z-20 left-0"
+          :style="filtersBox ? { height: '97%' } : { 'pointer-events': 'none' }"
+        >
+          <transition>
+            <div class="w-full h-full flex" v-if="filtersBox">
+              <div
+                class="h-full flex flex-col bg-white rounded-tr-xl rounded-br-xl filters-shadow overflow-y-auto overflow-x-hidden relative"
+                style="width: 90%"
+              >
+                <!-- Header -->
+                <div
+                  class="flex w-full flex-col bg-white md:sticky md:top-0 md:z-10"
+                  ref="FiltersHeader"
+                >
+                  <div
+                    class="flex flex-col w-full h-full"
+                    style="backdrop-filter: blur(1.5px)"
+                  >
+                    <!-- Title and info-->
+                    <div class="flex text-center mt-1">
+                      <h3 class="mx-0 px-2">
+                        Map of Memorials & Sites of Interest
+                      </h3>
+                    </div>
+
+                    <!-- Display number of active witches. -->
```

Comment:
comment should say locations here I think

Reply:
done